adoc/architecture-description.adoc

```diff
@@ -201,6 +201,106 @@ consumes this service, the kube proxy is the responsible for setting
+tasks and inserted into the main server. Filters have already been written
+to support various tasks such as raw TCP proxy, HTTP proxy, TLS client
+certificate authentication, etc.
+
+HTTP is such a critical component of modern application architectures that
+Envoy supports an additional HTTP L7 filter layer. HTTP filters can be
+plugged into the HTTP connection management subsystem that perform different
+tasks such as buffering, rate limiting, routing/forwarding, etc.
+
+When operating in HTTP mode, Envoy supports both HTTP/1.1 and HTTP/2. Envoy
+can operate as a transparent HTTP/1.1 to HTTP/2 proxy in both directions.
+This means that any combination of HTTP/1.1 and HTTP/2 clients and target
+servers can be bridged. The recommended service to service configuration uses
+HTTP/2 between all Envoys to create a mesh of persistent connections that
+requests and responses can be multiplexed over.
+
+When operating in HTTP mode, Envoy supports a routing subsystem that is
+capable of routing and redirecting requests based on path, authority, content
+type, runtime values, etc. This functionality is most useful when using Envoy
+as a front/edge proxy but is also leveraged when building a service to service
+mesh.
+
+gRPC is an RPC framework from Google that uses HTTP/2 as the underlying
+multiplexed transport. Envoy supports all of the HTTP/2 features required to
+be used as the routing and load balancing substrate for gRPC requests and
+responses. The two systems are very complementary.
+
+Load balancing among different components in a distributed system is a complex
+problem. Because Envoy is a self contained proxy instead of a library, it is
+able to implement advanced load balancing techniques in a single place and
+have them be accessible to any application. Currently Envoy includes support
+for automatic retries, circuit breaking, global rate limiting via an external
+rate limiting service, request shadowing, and outlier detection.
+
+Envoy is primarily designed as a service to service communication system, but
+there is benefit in using the same software at the edge (observability,
+management, identical service discovery and load balancing algorithms, etc.).
+Envoy includes enough features to make it usable as an edge proxy for most
+modern web application use cases. This includes TLS termination, HTTP/1.1 and
+HTTP/2 support, as well as HTTP L7 routing.
+
+The primary goal of Envoy is to make the network transparent.
+
+Cilium 1.6 supports Envoy proxy.
+
+* Cilium KVStore free operation
++
+The addition of a new CRD-based backend for security identities now allows to
+operate Cilium entirely without a KVstore in the context of Kubernetes.
+
+* Cilium 100% Kube-proxy replacement
++
+Operating a Kubernetes cluster without requiring to run kube-proxy has been a
+desire of many users. This release Cilium 1.6 includes the final two features
+required to run a Kubernetes cluster without kube-proxy with Cilium fully
+replacing kube-proxy.
```

Comment:
Careful, we haven't done the kube-proxy replacement in CaaSP. We will wait for SLE 15 SP2

Reply:
Done.